### DISCUSSION TITLE
Different color Div tags <div> : need this extension  . if something like this exists please provide me the name of extension .

### DISCUSSION THREAD

**Discussion by Maharshibhatnagar** [microsoft/vscode-discussions#702] [Extension Development QnA]
Suggestions from my side

**Comment by olguzzar** [comment]
Hi, you could try the [Rainbow Tags extension](https://marketplace.visualstudio.com/items?itemName=voldemortensen.rainbow-tags). As a note, this forum is for developing VS Code extensions, so I am going to close this discussion out. Thanks!
